rasa/core/processor.py

```diff
@@ -713,10 +713,23 @@ def _has_session_expired(self, tracker: DialogueStateTracker) -> bool:
 
         return has_expired
 
-    def _get_tracker(self, sender_id: Text) -> Optional[DialogueStateTracker]:
-        sender_id = sender_id or UserMessage.DEFAULT_SENDER_ID
+    def get_tracker(self, conversation_id: Text) -> Optional[DialogueStateTracker]:
+        """Get the tracker for a conversation.
```

Comment:
should we make this a `processor` method, so the contrast with `get_tracker_with_session_start()` is more apparent?

Reply:
Mhm, it's already a `processor` method? I could move it up so that they are right next to each other?